pymc3/variational/updates.py

```diff
@@ -344,7 +344,7 @@ def apply_nesterov_momentum(updates, params=None, momentum=0.9):
     updates : OrderedDict
         A dictionary mapping parameters to update expressions
     params : iterable of shared variables, optional
-        The variables to apply momentum to. If omitted, will apply
+        The variables to apply momentum too. If omitted, will apply
```

Comment:
Also here 😄

```suggestion
        The variables to apply momentum to. If omitted, will apply
```

Reply:
You are a meticulous person, @eigenfoo :smile:!